runtime/vm/jvminit.c

```diff
@@ -2606,9 +2606,6 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 					goto _memParseError;
 				}
 			}
-		} else {
-			/* default behavior (jvm disable HWPrefetch) */
-			setHWPrefetch(XXSETHWPREFETCH_NONE_VALUE);
```

Comment:
Should this set XXSETHWPREFETCH_OS_DEFAULT_VALUE instead?

Reply:
no.  running starts with OS-Default already.